src/connectedk8s/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+1.6.9
```

Comment:
version number should be 1.7.0 as new parameter added
please follow the bot suggest

Reply:
@AllyW I updated the PR accordingly, please take another look!